docs-chef-io/content/server/upgrades.md

```diff
@@ -471,6 +505,22 @@ The following External PostgreSQL upgrade steps are provided as a courtesy only.
    reindexdb: reindexing database "template1"
    ```
 
+1. Run `ANALYZE` on the PostgreSQL database.This process gathers statistics for the query planner to create the most efficient query execution paths. Accurate statistics will help the planner to choose the most appropriate query plan, and thereby improve the speed of query processing.
+    ```bash
+       /usr/bin/vacuumdb --all --analyze-only
+    ```
+
+   You should then see output like:
+   
+    ```bash
+       vacuumdb: vacuuming database "bifrost"
+       vacuumdb: vacuuming database "oc_id"
+       vacuumdb: vacuuming database "opscode-pgsql"
+       vacuumdb: vacuuming database "opscode_chef"
+       vacuumdb: vacuuming database "postgres"
+       vacuumdb: vacuuming database "template1"
+    ```
```

Comment:
Is there a section that links the commands to vacuum before this analyze? (I think this is the external PG section)

Reply:
You mean just a link to Vacuum command ?

Reply:
yes somewhere wee should mention even for external postgres that vacuum should be performed before running analyze